### ISSUE TITLE
 Activation function

### ISSUE THREAD

**Issue by wujiayi** [milesial/Pytorch-UNet#74]
Dear author,

Many thanks for sharing. I have a question about the activation loss function after the last convolutional layer. Why don't you use the  activation loss function after the last convolutional layer?

**Comment by wujiayi** [comment]
Thanks for your reply. I missed the F.sigmoid function in the u-net model. So sorry for this.
